test/unit/LoopbackModelBase.test.js

```diff
@@ -0,0 +1,48 @@
+const { expect } = require('chai');
+
+const LoopbackModelBase = require('../../src/LoopbackModelBase');
+
+const MockApp = require('../support/MockApp');
+
+describe('The LoopbackModelBase Class', function() {
+    it('throws an error if no valid loopback model is given', function(){
```

Comment:
Just out of curiosity: it throws an error if no models are given? (doesn't matter if it is a _valid_ loopback model?